custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -127,11 +127,14 @@ def _detect_model_artifact_file(self):
                 model_artifact_file = path
 
         if not model_artifact_file:
+            files_list = os.listdir(self._model_dir)
+            files_list_str = " | ".join(files_list)
             raise DrumCommonException(
                 "Could not find model artifact file in: {} supported by default predictors. "
                 "They support filenames with the following extensions {}. "
-                "If your artifact is not supported by default predictor, implement custom.load_model hook".format(
-                    self._model_dir, list(all_supported_extensions)
+                "If your artifact is not supported by default predictor, implement custom.load_model hook. "
+                "List of files got here are: {}".format(
+                    self._model_dir, list(all_supported_extensions), files_list_str
```

Comment:
I just made tihs change

Reply:
:+1: changes are in `master`?

Reply:
please don't use this PR anymore
I created a new one, rebased on master.

Reply:
master has changed